datastore/google/cloud/datastore/helpers.py

```diff
@@ -19,17 +19,18 @@
 
 import datetime
 import itertools
-
-from google.protobuf import struct_pb2
-from google.type import latlng_pb2
 import six
 
 from google.cloud._helpers import _datetime_to_pb_timestamp
 from google.cloud._helpers import _pb_timestamp_to_datetime
-from google.cloud.datastore_v1.proto import entity_pb2 as _entity_pb2
+from google.cloud.datastore_v1.proto import datastore_pb2
+from google.cloud.datastore_v1.proto import entity_pb2
 from google.cloud.datastore.entity import Entity
 from google.cloud.datastore.key import Key
 
+from google.protobuf import struct_pb2
+from google.type import latlng_pb2
```

Comment:
@chemelnucfin See PEP8 for import order (also, this is a cosmetic change unrelated to your PR).

Import sections should be (in order):

- Standard Library
- Third Party (i.e. from PyPI)
- First Party (i.e. from our own package)

The `google.protobuf` import is from [`protobuf`][1], the `google.type` import is from [`googleapis-common-protos`][2]. Hence they both belong in 3rd-party imports with `six`.

(I'm also sorry our `lint` tooling doesn't enforce this. Being a human linter is no fun for the reviewer or author.) 

[1]: https://pypi.org/project/protobuf/
[2]: https://pypi.org/project/googleapis-common-protos